src/services/__tests__/messagesService.test.ts

```diff
@@ -266,6 +297,58 @@ describe("MessageService#getMessagesByUser", () => {
   });
 });
 
+describe("MessageService#getServicesByUser", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("returns a list of services from the API", async () => {
+    mockGetServicesByUser.mockImplementation(() => {
+      return validApiServicesResponse;
+    });
+
+    const service = new MessageService(api);
+
+    const res = await service.getServicesByUser(mockedUser);
+
+    expect(mockGetClient).toHaveBeenCalledWith(mockedUser.fiscal_code);
+    expect(mockGetServicesByUser).toHaveBeenCalledWith(mockedUser.fiscal_code);
+    expect(res).toEqual(right(proxyServicesResponse));
+  });
+
+  it("returns an error if the getServicesByUser API returns an error", async () => {
+    mockGetServicesByUser.mockImplementation(() => {
+      return problemJson;
+    });
+
+    const service = new MessageService(api);
+
+    try {
+      await service.getServicesByUser(mockedUser);
+    } catch (e) {
```

Comment:
if `getServicesByUser `does not throw the following expect calls would be skipped and the test will pass. you have to call `expect.assertions(3)` before the try / catch block to check that the three expectations will be called

Reply:
Ok, I'll do it.